service/src/main/java/com/epam/esm/exceptions/GiftCertificateNotFoundException.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm.exceptions;
+
+public class GiftCertificateNotFoundException extends RuntimeException {
```

Comment:
Why do you extend exceptions from RuntimeException?

Reply:
They are handled by @ControllerAdvice so I thought I shouldn't try-catch them.